README.md

```diff
@@ -63,7 +63,7 @@ rustup install nightly
 
 ```shell
 # Ubuntu
-sudo apt install build-essential clang ocl-icd-opencl-dev libssl-dev
+sudo apt install build-essential clang libssl-dev
 
 # Archlinux
 sudo pacman -S base-devel clang ocl-icd openssl
```

Comment:
And `ocl-icd` from here?

Reply:
For whatever reason, removing cmake and opencl causes it to break. I'm not sure if these changes are doing anything or if we can actually remove these deps. @lemmih can you take a look?

Reply:
It might be required for testing but I'm not sure where it is in the codebase. I'm going to re-add the deps to the dockerfile and yaml

Reply:
Looks like `opencl` is required for `filecoin-proofs` https://github.com/filecoin-project/rust-fil-proofs